update_twiki.py

```diff
@@ -67,7 +69,9 @@ def insert_update(twiki_request, campaign, cross_section, frac_neg_wgts, target_
                     twiki_request['prepid'].split('-')[0],
                     cross_section,
                     frac_neg_wgts,
-                    target_num_events])
+                    target_num_events,
+                    0,
+                    'Please fill in with your comments'])
```

Comment:
I think it would be a better solution to add an empty string here and use `placeholder` attribute of `<textarea>` to display this text. We don't need to save "Please fill in with your comments" for each sample. We just want to show this message to a user.